docs/iceberg-rest-service.md

```diff
@@ -12,62 +12,75 @@ The Apache Gravitino Iceberg REST Server follows the [Apache Iceberg REST API sp
+|-------------------------|------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| How to build package    | `./gradlew compileIcebergRESTServer -x test` build a standalone Iceberg REST server in `distribution/gravitino-iceberg-rest-server` directory. | `./gradlew compileDistribution -x test` build Gravitino server and Iceberg REST server in `distribution/package` directory.       | 
+| How to assemble package | `./gradlew assembleIcebergRESTServer -x test` build a tar file with Iceberg REST server only in `distribution` directory.                      | `./gradlew assembleDistribution -x test` build a tar file with Gravitino server and Iceberg REST server `distribution` directory. | 
+| How to start server     | `./bin/gravitino-iceberg-rest-server.sh start`                                                                                                 | `./bin/gravitino.sh start`                                                                                                        | 
+| Configuration files     | `./conf/gravitino-iceberg-rest-server.conf`, `conf/log4j2.properties`                                                                          | `./conf/gravitino.conf`, `conf/log4j2.properties`                                                                                 | 
 
-2. [**Gravitino Iceberg metrics store Configuration**](#iceberg-metrics-store-configuration): you could implement a custom Iceberg metrics store and set corresponding configuration.
+You could modify environment variables in `conf/gravitino-env.sh` and check the log files in `./log/gravitino-iceberg-rest-server.log`, `./log/gravitino-iceberg-rest-server.out` for both mode.
 
-3. [**Gravitino Iceberg Catalog backend Configuration**](#gravitino-iceberg-catalog-backend-configuration): you have the option to set the specified catalog-backend to either `jdbc` or `hive`.
+## Apache Gravitino Iceberg REST catalog server configuration
 
-4. [**Other Iceberg Catalog Properties Defined by Apache Iceberg**](#other-apache-iceberg-catalog-properties): allows you to configure additional properties defined by Apache Iceberg.
+The configuration settings for the Gravitino Iceberg REST server are identical for both the standalone server and the auxiliary server in the Gravitino system. Starting from version `0.6.0`, using the prefix `gravitino.auxService.iceberg-rest.` for the auxiliary server configuration is deprecated. If both `gravitino.auxService.iceberg-rest.key` and `gravitino.iceberg-rest.key` are present, the `gravitino.iceberg-rest.key` will take precedence. The following configurations are shown with `gravitino.iceberg-rest.` prefix.
```

Comment:
"The Iceberg REST server configurations are the same in Gravitino for both standalone and auxiliary server mode..."

Reply:
done